docs/specs/app-initiated-sso-to-web.md

```diff
@@ -0,0 +1,179 @@
+&requested_token_type=urn%3Aauthgear%3Aparams%3Aoauth%3Atoken-type%3Adevice-browser-session-token
+```
+
+### Token Exchange Response
+
+The Token Exchange response for this profile has the following characteristics:
+
+- `device_secret`: Required. A new device secret must be returned. The mobile app should update its device secret with the new secret.
+- `x_device_browser_session_token`: Required. The device browser session token which can then be used to exchange for a browser cookie.
+- `expires_in`: Required. When the device browser session token expires.
+
+Here is an example:
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+{
+  "device_secret":"casdfgarfgasdfg",
+  "x_device_browser_session_token": "zxczxczxc",
+  "expires_in":300
+}
+```
+
+## Exchanging for browser cookie of a session with the authorization endpoint
+
+A web browser can open the /authorize endpoint to exchange for a valid browser session cookie with the following parameters:
+
+- `client_id`: Required. The client_id of the obtained session.
+- `id_token_hint`: Required. The id_token obtained by the mobile app.
+- `x_device_browser_session_token`: Required. The device browser session token obtained from token exchange response.
+- `scope`: Optional. Scope of the session of the browser session cookie. If not provided, it will be same as the scope of the session which generates this browser session cookie (i.e. The offline grant of the mobile app).
+- `prompt`: Optional. If provided, must be `none`.
+- `response_type`: Required. Must be one of the following values:
+  - `session`: A long lived token will be set in the cookie, which shares the same lifetime of the refresh_token. However, it will not live longer than an IDP session.
+  - `token`: A short lived token will be set in the cookie, which shares the same lifetime of an access_token.
+- `response_mode`: Required. Must be `cookie`.
+- `redirect_uri`: Required. The uri to redirect to on the operation completed. It must be whitelisted by the client specified in `client_id`.
+- `state`: Optional. If provided, will be provided to the redirect_uri as a `state` query parameter.
+
+In the response, the authorize endpoint will redirect the user to `redirect_uri`, and set a browser cookie.
+
+## Security Considerations
+
+### Binding Tokens To Device
+
+device_secret and x_device_browser_session_token should always be bound to a device.
+
+[OAuth 2.0 Demonstrating Proof of Possession](https://datatracker.ietf.org/doc/html/rfc9449) is implemented for such binding.
+
+The DPoP Proof MAY be provided when making requests to the /token endpoint. If DPoP Proof is provided, device_secret and x_device_browser_session_token returned in the response will be bound to the public key provided in the DPoP Proof.
+
+## Changes In SDKs
+
+- `authenticate()` will accept a new option `isSSOToWebEnabled: boolean`. When `true`, the `x_device_browser_sso` will be added to the authorize request.
```

Comment:
`isAppInitiatedSSOToWebEnabled`. Stay consistent with the term.

Reply:
As the sdk is in the app, mentioning "AppInitiated" looks a bit redundant, but I understand that it is easier to for the user to find the config if they have the same name.